### DISCUSSION TITLE
Blocking google.com search

### DISCUSSION THREAD

**Discussion by MateoIs74** [SelfControlApp/selfcontrol#809] [Q&A]
Hello

I've seen few post about blocking Google.com and all sub domains, however I'd like to block the google search url and still be able to the use the suite of apps within https://accounts.google.com/ (such as gmail, calendar, drive, etc). Is this possible? I tried blocking http://images.google.com as a test and that seems to work, but I'm concerned that if I add google.com to the block list, that it will block everything. I have a long blocker currently running, so I can't test this. 

Thanks

**Comment by aungKhantPaing** [comment]
No, it won't affect other domains like accounts.google.com. I can use music.youtube.com while blocking youtube.com. It'll also be the same for google.com.
